src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt

```diff
@@ -96,6 +97,7 @@ class SpoofSignatureVerificationPatch : BytecodePatch(
         } ?: return OpenCronetDataSourceFingerprint.toErrorResult()
 
         // hook override subtitles
+        if (false) { // Subtitle fix is not required for current protobuf parameter spoofing
```

Comment:
The following code should be removed, including the integrations. Git saves this in history if we need this anytime again.